sympy/series/tests/test_limits.py

```diff
@@ -608,3 +608,7 @@ def test_issue_17671():
 
 def test_issue_18306():
     assert limit(sin(sqrt(x))/sqrt(sin(x)), x, 0, '+') == 1
+
+def test_issue_18442():
+    assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == \
+            AccumBounds(-oo, oo)*AccumBounds(-oo, oo)**(2**(sqrt(pi)))
```

Comment:
I think that this could be simply `AccumBounds(-oo, oo)`. That is one way of saying that there is no limit.

Reply:
It is getting simplified to `AccumBounds(-oo, oo)**(1 + 2**(sqrt(pi)))` using
`limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-').simplify()`

Reply:
In this branch, I get without any simplifications:
```
>>> from sympy import *
>>> x = Symbol('x')
>>> limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
AccumBounds(-oo, oo)
```

Reply:
I mistakenly checked on the branch without last commit. Thanks :)